instantapp/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.clevertap.demo"
```

Comment:
why is this removed, does it work properly without it?

Reply:
Yess. Namespace is now needed in the gradle file and not the manifest file